src/mds/MDSMap.h

```diff
@@ -416,72 +422,6 @@ class MDSMap {
 	s.insert(p->second.rank);
   } 
 
-  int get_random_up_mds() {
```

Comment:
Why'd you zap this one? It just got transplanted into Client.cc and it is an mds-cluster-specific thing.